### ISSUE TITLE
Support  for other Xiaomi Devices

### ISSUE THREAD

**Issue by Tobix99** [mKeRix/room-assistant#674]
**Is your feature request related to a problem? Please describe.**
Hey, so I've your Software (which is awesome btw) running on a RPi Zero W. And it works great with the iOS App and the iBeacons, but i'm having trouble with this seconds [docker](https://github.com/lolouk44/xiaomi_mi_scale), which is taking the BLE Packets from my Mi Scale (XMTZC05HM) and publishes it to my MQTT Broker. They both sometimes fight about the Bluetooth Adapter.

**Describe the solution you'd like**
So there are already Mi BLE Devices implemented. Maybe it's not that hard to add this scale too, but i'm not sure. Maybe if I get some free time I will look into it and try to add it via a PR. How hard can BLE be :P

**Comment by mKeRix** [comment]
Yeah, room-assistant can be quite bossy with the adapter - that's unfortunately required to work around some temporary failures that may appear. The code that you linked should be possible to implement in room-assistant as well, at least the necessary advertisement data should already be available. Unfortunately, I don't own such a scale so I can't confidently build this myself. I'm happy to help with contributions by answering questions and reviewing though! :)

Maybe #458 could also be a workaround for this when that is implemented.
